packages/react-scripts/template/README.md

```diff
@@ -1479,6 +1480,73 @@ If you use [Visual Studio Code](https://code.visualstudio.com), there is a [Jest
 
 ![VS Code Jest Preview](https://cloud.githubusercontent.com/assets/49038/20795349/a032308a-b7c8-11e6-9b34-7eeac781003f.png)
 
+## Debugging Tests
+
+There are various ways to setup a debugger for your Jest tests. We cover debugging in Chrome and Visual Studio Code.
+
+### Chrome & Node Inspector
+
+Download [Node Inspector](https://github.com/node-inspector/node-inspector) or similar debugger
+
+```bash
+npm install -g node-inspector
+```
+
+Place `debugger;` statements in any test and run
+
+macOS/Linux
+```bash
+node node_modules/.bin/react-scripts --inspect-brk test --runInBand --env=jsdom
+```
+
+Windows
+```bash
+node node_modules/bin/react-scripts --inspect-brk test --runInBand --testPathIgnorePatterns=ui-tests/* --env=jsdom
+```
+
+This will start running your Jest tests, but pause before executing to allow a debugger to attach to the process.
+
+To attach the `node-inspector` debugger, run:
+```
+node-inspector
```

Comment:
I'm a bit out of touch but I had an impression that `node-inspector` is no longer necessary, and Node integration just works out of the box with Chrome. Is that wrong?

Reply:
Nice! I had no idea, just gave it a shot and you are correct. 

Updating this!